src/unix/glegl.cpp

```diff
@@ -628,11 +615,26 @@ EGLConfig *wxGLCanvasEGL::InitConfig(const wxGLAttributes& dispAttrs)
         wxFAIL_MSG("Unable to get EGL Display");
         return nullptr;
     }
-    if ( !eglInitialize(dpy, nullptr, nullptr) )
+
+    EGLint egl_major = 0;
+    EGLint egl_minor = 0;
+    if ( !eglInitialize(dpy, &egl_major, &egl_minor) )
     {
         wxFAIL_MSG("eglInitialize failed");
         return nullptr;
     }
+
+    // The runtime EGL version cannot be known until EGL has been initialized.
+    if ( egl_major < 1 || (egl_major == 1 && egl_minor < 5) )
+    {
+        // Ignore the return value here, we cannot recover at this point.
+        eglTerminate(dpy);
+        wxFAIL_MSG(wxString::Format(
+            "EGL version is %d.%d. EGL version 1.5 or greater is required.",
+            egl_major, egl_minor));
```

Comment:
I wonder if this shouldn't be `wxLogError()`, so that it is reported to the end user in production builds (where `wxFAIL_MSG()` does nothing by default), as it might be something they can fix or at least tell the developer about.

Reply:
I changed it. I think it's going to be more useful down the line, when we can fallback more graciously for older versions. Anyway it doesn't change much right now since we end up calling `wxFAIL_MSG()` down the line.